index.html

```diff
@@ -130,6 +137,44 @@ <h1 class="text-center">About Us</h1>
     </section>
     <!-- about ends -->
 
+    
+<!--issue.104 starts-->
+<section id="issue.104" class="issue.104">
+<div class="container">
+    <h1 class="text-center">Sharpen Your Skills</h1>
+    <hr>
+<br><br>
+<table class="center">
+  <tr>
+    <th><i class="fas fa-atom"></i>&nbsp; <a href="  ">Data Structures</a></th>
+    <th><i class="fas fa-project-diagram"></i>&nbsp; <a href="  ">Algorithms</a></th>
+    <th><i class="fab fa-python"></i>&nbsp; <a href="">Python</a></th>
+  </tr>
+  <tr>
+     <th><i class="fab fa-cuttlefish"></i>&nbsp; <a href="  ">C</a></th>
+    <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="  ">C++</a></th>
+    <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="  ">Java</a></th>
+  </tr>
+  <tr>
+    <th><i class="fas fa-gem"></i>&nbsp; <a href="  ">Ruby</a></th>
+    <th><i class="fas fa-square-root-alt"></i>&nbsp; <a href="  ">Mathematics</a></th>
+    <th><i class="fa fa-database"></i>&nbsp; <a href="  ">Databases</a></th>
+  </tr>
+  </table>
+
+</div>
+</section>
+<p1></p1>
+    <!--issue.104 ends-->
+
```

Comment:
A single empty line is enough to differentiate the section remove rest all blank lines.

Reply:
done